components/aws/sagemaker/process/README.md

```diff
@@ -13,29 +12,69 @@ Argument        | Description                 | Optional   | Data type  | Accept
-container_entrypoint | The entrypoint for the processing job. This is in the form of a list of strings that make a command | Yes | Yes | List of Dicts | | [] |
-container_arguments | A list of string arguments to be passed to a processing job | Yes | Yes | List of Dicts | | [] |
-input_config | Parameters that specify Amazon S3 inputs for a processing job | No | Dict | | {} |
-output_config | Parameters that specify Amazon S3 outputs for a processing job | No | Dict | | {} |
+container_entrypoint | The entrypoint for the processing job. This is in the form of a list of strings that make a command | Yes | Yes | List of Strings | | [] |
+container_arguments | A list of string arguments to be passed to a processing job | Yes | Yes | List of Strings | | [] |
+input_config | Parameters that specify Amazon S3 inputs for a processing job | No | List of Dicts | | [] |
+output_config | Parameters that specify Amazon S3 outputs for a processing job | No | List of Dict | | [] |
 vpc_security_group_ids | A comma-delimited list of security group IDs, in the form sg-xxxxxxxx | Yes | String | | |
 vpc_subnets | A comma-delimited list of subnet IDs in the VPC to which you want to connect your hpo job | Yes | String | | |
-network_isolation | Isolates the training container if true | No | Boolean | False, True | True |
-traffic_encryption | Encrypts all communications between ML compute instances in distributed training if true | No | Boolean | False, True | False |
+network_isolation | Isolates the processing container if true | No | Boolean | False, True | True |
+traffic_encryption | Encrypts all communications between ML compute instances in distributed processing if true | No | Boolean | False, True | False |
 tags | Key-value pairs to categorize AWS resources | Yes | Dict | | {} |
 
+Notes:
+* You can find more information about how container entrypoint and arguments are used at the [Build Your Own Processing Container](https://docs.aws.amazon.com/sagemaker/latest/dg/build-your-own-processing-container.html#byoc-run-image) documentation.
+* Each key and value in the `environment` parameter string to string map can have length of up to 1024. SageMaker supports up to 16 entries in the map.
+* The format for the `input_config` field is:
+```
+[
+  {
+    'InputName': 'string',
+    'S3Input': {
+      'S3Uri': 'string',
+      'LocalPath': 'string',
+      'S3DataType': 'ManifestFile'|'S3Prefix',
+      'S3InputMode': 'Pipe'|'File',
+      'S3DataDistributionType': 'FullyReplicated'|'ShardedByS3Key',
+      'S3CompressionType': 'None'|'Gzip'
+    }
+  },
+]
+```
+* The format for the `output_config` field is:
+```
+[
+  {
+    'OutputName': 'string',
+    'S3Output': {
+      'S3Uri': 'string',
+      'LocalPath': 'string',
+      'S3UploadMode': 'Continuous'|'EndOfJob'
+    }
+  },
+]
+```
+
+## Outputs
+Name | Description
+:--- | :----------
+job_name | Processing job name
+output_artifacts | A dictionary mapping output name to output S3 URI
```

Comment:
nit: A dictionary mapping with output_config's `OutputName` as key and `S3Uri` as values